akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventsourcedRunning.scala

```diff
@@ -197,6 +200,9 @@ import scala.collection.immutable
       }
     }
 
+    override def onSignal: PartialFunction[Signal, Behavior[InternalProtocol]] =
+      { case signal ⇒ returnPermitOnStop((setup.context, signal)) }
```

Comment:
not needed

Reply:
Right thanks; This exposed potential need for docs in case someone wants to share such handler which I added as todo to the guide https://github.com/akka/akka/issues/22805 :)